### DISCUSSION TITLE
为什么下载dash视频的时候会出现416错误？

### DISCUSSION THREAD

**Discussion by 1357310795** [the1812/Bilibili-Evolved#2074] [综合讨论 / General]
![111221](https://user-images.githubusercontent.com/35869804/126861200-718313d2-08a9-4716-aa6a-e72862ed3779.png)

### 下载到最后卡住了，白白浪费硬盘写入寿命。。。

想知道416错误是B站的问题还是脚本的问题？
为什么不能重新尝试获取请求失败的视频片段？

**Comment by the1812** [comment]
脚本的问题, 换下面的导出吧
另外没下完之前都在内存里, 不会写入硬盘

**Reply by 1357310795** [reply]
那么重新尝试请求是否可行呢？

**Reply by the1812** [reply]
416是视频长度计算错误了, 再请求也还是416的
